common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -49,7 +49,49 @@ class SequenceFields(object):
     )
 
 
-class SequenceModule(SequenceFields, XModule):
+class ProctoringFields(object):
+    """
+    Fields that are specific to Proctored or Timed Exams
+    """
+    is_time_limited = Boolean(
+        display_name=_("Is Time Limited"),
+        help=_(
+            "This setting indicates whether students have a limited time"
+            " to view or interact with this courseware component."
+        ),
+        default=False,
+        scope=Scope.settings,
+    )
+
+    default_time_limit_mins = Integer(
+        display_name=_("Time Limit in Minutes"),
+        help=_(
+            "The number of minutes available to users for viewing or interacting with this courseware component."
+        ),
+        default=None,
+        scope=Scope.settings,
+    )
+
+    is_proctored_enabled = Boolean(
+        display_name=_("Is Proctoring Enabled"),
+        help=_(
+            "This setting indicates whether this exam is a proctored exam."
+        ),
+        default=False,
+        scope=Scope.settings,
+    )
+
+    is_practice_exam = Boolean(
+        display_name=_("Is Practice Exam"),
+        help=_(
+            "This setting indicates whether this exam is only test that will not be fully verified."
```

Comment:
"This setting indicates whether this exam is used for testing purposes only. Practice exams are not verified."
(@chrisdodge Is this still accurate if we say "not verified" vs "not fully verified."?)